### ISSUE TITLE
AAChartView背景色有色偏

### ISSUE THREAD

**Issue by silendir** [AAChartModel/AAChartKit#1010]
#141414的背景.. 需要 .backgroundColorSet(@"#191919") 才能融合
![IMG_1170](https://user-images.githubusercontent.com/11792633/94535258-f3d13200-0273-11eb-9b19-e6c4a34e401e.PNG)

**Comment by silendir** [comment]
![IMG_1170](https://user-images.githubusercontent.com/11792633/94538355-81158600-0276-11eb-9a49-690fc7d33a20.PNG)

**Comment by silendir** [comment]
![截屏2020-09-29 下午5 09 48](https://user-images.githubusercontent.com/11792633/94538424-9b4f6400-0276-11eb-97a6-1f64bc995a88.jpg)

**Comment by silendir** [comment]
而且画图的时候AAChartView会闪一下白

**Comment by AAChartModel** [comment]
为啥要拍照呢?直接电脑上截图不好吗?这画面有点糊啊

**Comment by silendir** [comment]
鼠标移开捕捉不了测色计

**Comment by AAChartModel** [comment]
<img width="135" alt="截屏2020-09-29 下午5 55 42" src="https://user-images.githubusercontent.com/16357599/94543721-000dbd00-027d-11eb-84b9-3913e58a5e76.png">

你这里的小图表都是 AAChartView?

**Comment by silendir** [comment]
是的是不是有点蠢, 发现有点慢, 黑色背景在首次实例化 的时候会闪一下白屏, 不知道有没有特别针对这种缩略图的, UI 要还原曲线

**Comment by AAChartModel** [comment]
主要是 AAChartKit 有点蠢🤣, 因为 AAChartView 继承自 WKWebView, WKWebView几乎就相当于一个浏览器,是很重的 UI 视图控件, 同一个展示多个WKWebView性能可想而知,
这个最好还是用原生的来画比较好

**Comment by AAChartModel** [comment]
所有 Objective-C 语言的图表

https://github.com/search?o=desc&q=chart+language%3Aobjc&s=stars&type=Repositories

**Comment by AAChartModel** [comment]
所有 Swift 语言的图表
https://github.com/search?o=desc&q=chart+language%3Aswift&s=stars&type=Repositories

**Comment by silendir** [comment]
我知道webView,一般我用textView代替webView

**Comment by silendir** [comment]
不管他, 懒的搞, 8以上没压力😝

**Comment by AAChartModel** [comment]
又不是不能用🤣🤣🤣

**Comment by AAChartModel** [comment]
但是从环保的角度讲,还是用原生的好些哈哈哈😁😄😄

**Comment by silendir** [comment]
等他们说卡的时候再改,免得等一下说款式不对应

**Comment by silendir** [comment]
能获取到h5 的代码直接替换其他view么

**Comment by AAChartModel** [comment]
你们的的这个 APP 怎么搞得像是专门画图表的, 各种需求都挺特别,有点难搞

**Comment by silendir** [comment]
迟点还有音频和波形的处理,FinalCut级别的 ..

**Comment by silendir** [comment]
我拳打FFMpeg , 脚踢AAChartKit

**Comment by AAChartModel** [comment]
> 能获取到h5 的代码直接替换其他view么

哪有这么牛逼的黑科技, 真能够这样,那所有 web APP 岂不是都能自动转换为原生应用?

不过你这里如果都是静态的缩略小图,倒是可以用同一个 AAChartView 先去生成一组图表的图片,再将图表图片添加到 UITableView 上,这样性能上或许会好点

**Comment by silendir** [comment]
外面这个缩略图, 点进去要忠实还原里面的曲线....  要求很高的... 反正能实现.... 现在还是够用的

**Comment by AAChartModel** [comment]
> 倒是可以用同一个 AAChartView 先去生成一组图表的图片

只是不知道这个过程的耗时怎么样,没试过,有机会我自己测试一下看看

**Comment by AAChartModel** [comment]
> 我拳打FFMpeg , 脚踢AAChartKit

计算机图形科学大佬转行 iOS 开发,失敬失敬

**Comment by silendir** [comment]
是的, 只需要静态的缩略图,不需要互动ToolTip..我都屏蔽了, 应该可以直接drew到uiview上.不过懒得搞..等下又说角度不对太尖了....哎...懒得搞

**Comment by AAChartModel** [comment]
> 点进去要忠实还原里面的曲线

你这个是点进去进入详情页,应该还行我觉得

**Comment by silendir** [comment]
没有没有, 这里有好几组大佬, 有结构工程师,有视频处理, 有嵌入式, 有音乐的专门算法师, 我就是个递鞋的

**Comment by silendir** [comment]
对的, 就是外面一个"忠实还原" 的缩略图 (死的,可以没有动画) 然后点进去才是连续几个AAChartView这样

**Comment by silendir** [comment]
不过你这里如果都是静态的缩略小图,倒是可以用同一个 AAChartView 先去生成一组图表的图片,再将图表图片添加到 UITableView 上,这样性能上或许会好点

**Comment by silendir** [comment]
不过你这里如果都是静态的缩略小图,倒是可以用同一个 AAChartView 先去生成一组图表的图片,再将图表图片添加到 UITableView 上,这样性能上或许会好点

大佬的意思是..我复用一个生成画面,然后给View拍照的方式批处理一批死的图片对吧? 好像是个办法, 但是好像处理起来也要好几秒

**Comment by AAChartModel** [comment]
> 没有没有, 这里有好几组大佬, 有结构工程师,有视频处理, 有嵌入式, 有音乐的专门算法师, 我就是个递鞋的

你们这是要复刻 FinalCut 嘛,这么骚气? 贵公司还缺人不?给大佬递一杯阔落

**Comment by AAChartModel** [comment]
> 好像是个办法, 但是好像处理起来也要好几秒

我后面试试吧,如果还行的话就把它添加到 AAChartKit 的 demo 当中,充当新的 AAChartKit 使用示例

**Comment by silendir** [comment]
您才是老大, 我是渣渣, 我这点微薄工资不知道请不请得起您

**Comment by silendir** [comment]
我刚试了下, iPhone 7 需要延迟1秒,才能遮蔽5-10个闪烁, 用时 还是挺多的, 加上拍照,光加载默认动画那零点几秒肯定不够

**Comment by AAChartModel** [comment]
> 光加载默认动画那零点几秒肯定不够

如果在上一页就开始做这些准备工作呢?

**Comment by silendir** [comment]
我觉得app一启动就去做会比较现实, 我担心很多用户一进来就直接怼进来这里
